stack/indexer/deb/debian/postinst

```diff
@@ -14,12 +14,13 @@ export USER=${NAME}
 export GROUP=${NAME}
 export CONFIG_DIR=/etc/${NAME}
 export INSTALLATION_DIR=/usr/share/${NAME}
+export BACKUP_DIR=/usr/share/wazuh-indexer-backup
```

Comment:
Why can we use an internal folder? It's not recommended to go out of the wazuh-indexer folder scope.

Reply:
In the upgrade process, the `postrm remove` is executed, and in that stage, the installation directory is removed. A good alternative is to store the backup files inside de `CONFIG_DIR` directory, which is not removed in the `postrm remove` stage.

```
remove)
        REMOVE_DIRS=true
    ;;
```
```
if [ "$REMOVE_DIRS" = "true" ]; then
    if [ -d "${INSTALLATION_DIR}" ]; then
        echo -n "Deleting installation directory..."
        rm -rf "${INSTALLATION_DIR}" > /dev/null 2>&1
        echo " OK"
    fi
```
Test: https://github.com/davidcr01/wazuh-packages/actions/runs/3912482510/jobs/6687179305